### PR TITLE
Make wopO stop when value isn't starting with 0x

### PR DESCRIPTION
**Pull request by giuscri**
screenshot fee7309cb57b84718b28d6604b8009ad

Tests: https://github.com/radare/radare2-regressions/pull/1213

### CODE REVIEW

**Comment by giuscri** [comment]
@ret2libc, pushed new change

On Wed, Feb 28, 2018 at 4:06 PM, Riccardo Schirone <notifications@github.com

**Comment by ret2libc** [comment]
I was thinking... Do we really want to limit it to hex only? `r_num` handles multiple number representations already...

**Comment by xarkes** [comment]
Yes indeed, maybe a warning is enough, or at least format precision in the help message.

**Comment by giuscri** [comment]
Yes, because accepting other format is unexpected behaviour. See user that passed 41414142 and expect that to be interpreted as hex

**Comment by giuscri** [comment]
imho accepting other format is confusing (for the user; code for accepting other value formats _was_ already there before this pr)

File: binr/ragg2/ragg2.c [hunk 1]
```diff
@@ -315,9 +315,13 @@ int main(int argc, char **argv) {
 
 	// catch this first
 	if (get_offset) {
-		get_offset = r_num_math (0, sequence);
-		printf ("Little endian: %d\n", r_debruijn_offset (get_offset, false));
-		printf ("Big endian: %d\n", r_debruijn_offset (get_offset, true));
+		if (!strncmp (sequence, "0x", 2)) {
```
Comment (radare) [inline binr/ragg2/ragg2.c]: If there's an error it should return 1.

but actually, it should work with other bases.. but base16 is probably the most expected one and can solve some confussions.. i guess

**Pull request merged**